modules/domain-memory/sources/MemoryAggregateStore.kt

```diff
@@ -15,6 +16,10 @@ private class MemoryAggregateStore : RaptorAggregateStore {
 	}
 
 
+	override suspend fun lastEventTimestampOrNull(): Timestamp? =
+		events.maxOfOrNull { it.timestamp }
```

Comment:
It's inefficient to go through all events on every request. We can just use `events.lastOrNull()?.timestamp`  because they're ordered oldest to newest.